src/naip_cnn/config.py

```diff
@@ -1,3 +1,14 @@
+from pathlib import Path
+
+# The Earth Engine path where assets are stored
+EE_ASSET_DIR = "projects/ee-maptheforests/assets"
```

Comment:
Mostly just something to tidy up! The band names were lost when I ingested the images manually (if you know of any workaround to avoid that, let me know!), so I used a Code Editor script to re-set the band names and re-ingest them into a collection. It's a bit of a pain, but I'm assuming we're not going to be adding new training data frequently enough to worry about a better system.

Reply:
Makes sense to me!  (Sorry, I don't have anything off the top of my head to make band names stick.  I'm always impressed when *exported* TIFs from GEE maintain those band names, so surely there is a way to do it with GDAL, but I haven't researched this fully.)